src/app.js

```diff
@@ -155,14 +118,8 @@ export default function (app) {
   }
 
   function setElementData(element, name, value, oldValue) {
-    name = name.toLowerCase()
-
     if (name === "key") {
```

Comment:
Since this has heavy traffic in terms of ops, I would be really interested in seeing performance benchmarks with `switch` vs.`if` here 🎉